src/components/team/style.scss

```diff
@@ -0,0 +1,71 @@
+// todo continue styling
+@import "./src/styles/base/variables";
```

Comment:
enough `"styles/base/variables"` :)

Reply:
Not always - depends on your IDE :) WebStorm has error without dot :)